packages/blockchain/celo/src/lib/transactions/erc721/index.ts

```diff
@@ -583,50 +594,52 @@ const updateCashbackForAuthorSignedTransaction = async (
   testnet?: boolean,
 ) => {
   const { fromPrivateKey, cashbackValue, tokenId, contractAddress, feeCurrency, nonce, signatureId } = body
+  if (feeCurrency && contractAddress) {
+    const celoProvider = celoUtils.getProvider(provider)
+    const network = await celoProvider.ready
+    const feeCurrencyContractAddress = celoUtils.getFeeCurrency(feeCurrency, testnet)
+    const contract = new new Web3().eth.Contract(Erc721Token_Cashback.abi as any, contractAddress.trim())
```

Comment:
is it ok ```new new```?

Reply:
I dont know but it works ok :-D I didn't wrote that
![image](https://user-images.githubusercontent.com/10884869/184616147-c76ce34e-1600-4b1a-993f-bec3745b180f.png)